.github/ISSUE_TEMPLATE/01-question.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F4AC Question"
```

Comment:
Is it possible to use emoji notation like :speech_balloon:(`:speech_balloon:`)? It's easier to read and understand.

Reply:
Unfortunately no.
[The header part sandwiched by `---` is YAML format](https://help.github.com/en/enterprise/2.16/user/articles/manually-creating-a-single-issue-template-for-your-repository). So we can't use ":" character and it does not work even if it is double quoted.